torch/_dynamo/utils.py

```diff
@@ -398,6 +398,10 @@ def clone_tensor(x):
 
 def clone_input(x):
     """copy while preserving strides"""
+    # TODO: this is questionable
+    if isinstance(x, torch._subclasses.FakeTensor):
+        # this func fails on fake tensors in __torch_dispatch__
+        return x
```

Comment:
I'll put up with this because I think my pending refactor will resolve the confusion here